changelog/5714.bugfix.rst

```diff
@@ -0,0 +1 @@
+Only open one connection to the JSOC for any download.
```

Comment:
We should explain why we are making this change here (i.e. JSOC told us to do this), also we should make it clear that parameters are overriden, especially as we are backporting it as well.

Reply:
Changed.